visualization/app/codeCharta/ui/legendPanel/legendPanel.component.spec.ts

```diff
@@ -74,6 +70,35 @@ describe("LegendPanelController", () => {
 		expect(metricDescriptions[2].textContent).toMatch("Color metric: Real Lines of Code (rloc)")
 	})
 
+	it("should contain elements with titles and links if attributeDescriptors are present", async () => {
+		const { container, detectChanges } = await render(LegendPanelComponent, { excludeComponentDeclaration: true })
+		const store = TestBed.inject(MockStore)
+		const pipe = new AttributeDescriptorTooltipPipe()
+		const metricLink = "https://rl.oc"
+		store.overrideSelector(isDeltaStateSelector, false)
+		const mccAttributeDescriptor = {
+			mcc: { title: "MCC_Title", description: "MCC_description", hintLowValue: "MCC_lowValue", hintHighValue: "", link: "" }
+		}
+		store.overrideSelector(attributeDescriptorsSelector, {
+			...mccAttributeDescriptor,
+			rloc: { title: "RLOC_Title", description: "RLOC_Description", hintLowValue: "", hintHighValue: "", link: metricLink }
+		})
+		store.refreshState()
+		detectChanges()
+		fireEvent.click(screen.getByTitle("Show panel"))
+		const areDeltaEntriesShown = screen.queryAllByText("delta", { exact: false }).length > 0
+		expect(areDeltaEntriesShown).toBe(false)
```

Comment:
The following suggestion has the following advantages:
- no tmp var needed
- would fail with a jest dom error instead of a "expected true to be false"

```suggestion
		expect(screen.getAllByText("delta", { exact: false }).length).toBeGreaterThan(0)
```

Reply:
same goes for the other places where  screen.**query**AllByText("delta" is used